test-util/src/main/java/io/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -455,6 +459,35 @@ private StringBuilder summarizeRejection(final Record<?> record) {
         .append(record.getRejectionReason());
   }
 
+  private String summarizeTimer(final Record<?> record) {
+    final var value = (TimerRecordValue) record.getValue();
+    final var builder = new StringBuilder();
+    final var dueTime = Instant.ofEpochMilli(value.getDueDate()).atZone(ZoneId.systemDefault());
+
+    builder.append("@").append(formatId(value.getTargetElementId()));
+
+    if (value.getElementInstanceKey() > 0) {
+      builder.append("[").append(shortenKey(value.getElementInstanceKey())).append("]");
+    } else {
+      builder.append(" (start event)");
+    }
+
+    builder
+        .append(" in <process [")
+        .append(shortenKey(value.getProcessDefinitionKey()))
+        .append("]>");
+
+    if (value.getProcessInstanceKey() > 0) {
+      builder.append(" instance <[").append(shortenKey(value.getProcessInstanceKey())).append("]>");
+    }
```

Comment:
Following DRY, we should use `summarizeProcessInformation()` instead.

Reply:
Here we don't have the bpmnElementId but rather the process definition key. We could add an overload for this specific case I suppose, which would show:

- ` in <process K03[K05]> ` for normal catch events
- ` in <process K03[?]> ` for start events

Of course I can always format the key and pass that as the `bpmnElementId`, which will do the same, but may look strange in the future if we change how we format IDs :sweat_smile: Actually, thinking about it, it's probably fine for now to do that.